### PR TITLE
doc: releases: add MbedTLS-related notes

### PR DESCRIPTION
**Pull request by tomi-font**
For changes brought by PRs #71118 and #72078.

### CODE REVIEW

File: doc/releases/migration-guide-3.7.rst [hunk 1]
```diff
@@ -39,6 +39,12 @@ Boards
 Modules
 *******
 
+MbedTLS
+=======
+
+* The default enabling of the hash algorithms SHA-384, SHA-512, MD5 and SHA-1 has been removed.
```
Comment (aescolar) [inline doc/releases/migration-guide-3.7.rst:45]: Just a nit , the grammar is a bit funny.
```suggestion
* Hash algorithms SHA-384, SHA-512, MD5 and SHA-1 are not enabled by default anymore.
```

**Review by aescolar — COMMENTED** [review]
Thanks @tomi-font just a nit from me

File: doc/releases/migration-guide-3.7.rst (hunk 1 above)
Comment (tomi-font) [reply doc/releases/migration-guide-3.7.rst]: Haha I had it this way but changed it last second, seems like I shouldn't have.

**Review by aescolar — APPROVED** [review]

**Review by kartben — APPROVED** [review]
You may drop "retroactive" from the commit message, although I understand why you might have added it -- 3.7 isn't out yet so it's perfectly fine that release notes and migration guide still evolve :)

**Review by nashif — APPROVED** [review]

**Pull request merged**